.github/workflows/release.yaml

```diff
@@ -45,12 +43,44 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
 
-      - name: release
-        run: make release
```

Comment:
maybe we can remove the makefile rule for this `make release` to clean up things

Reply:
Done